### ISSUE TITLE
nim v1 failed test: test_int_muldiv

### ISSUE THREAD

**Issue by jangko** [status-im/nim-stint#88]
this test failed with Nim 1.0.0

```text
    F:\projects\nim-stint\tests\test_int_muldiv.nim(77, 32): Check failed: cast[int64](qr.rem) == -100'i64 mod 13
    -100'i64 mod 13 was 9
  [FAILED] Divmod(-100, 13) returns the correct result
    F:\projects\nim-stint\tests\test_int_muldiv.nim(86, 32): Check failed: cast[int64](qr.rem) == 100'i64 mod -13
    100'i64 mod -13 was -9
  [FAILED] Divmod(100, -13) returns the correct result
```

**Comment by jangko** [comment]
it is a Nim 0.20.0/1.0.0 bug nim-lang/Nim#12332

**Comment by arnetheduck** [comment]
> [1:54 PM] BOTgitterirc: <Araq> well regarding #12332, most probably the fix won't be backported

**Comment by jangko** [comment]
closing. fixed by nim-lang/Nim#12402
